### PR TITLE
Bug 1337328 : Added text mongodb variable

### PR DESCRIPTION
**Pull request by bfallonf**
@bparees @mfojtik As per email.

I've pretty much just added the variable to the table with a link to the text section of the Mongo docs. I don't think this warrants its own section explaining it. Let me know if you disagree.

Any other thoughts?

### CODE REVIEW

**Comment by mfojtik** [comment]
lgtm

**Comment by bparees** [comment]
needs a note that it's only applicable to mongodb 2.4 (mongodb 2.6 doesn't have an equivalent setting)

**Comment by bfallonf** [comment]
@mfojtik @bparees Cool. Thanks! I added a note in parenthesis.

If there's nothing else, @tpoitras @adellape @ahardin-rh @tnguyen-rh could probably have a read/merge/both.

**Comment by bparees** [comment]
lgtm

File: using_images/db_images/mongodb.adoc [hunk 1]
```diff
@@ -162,15 +162,19 @@ MongoDB settings can be configured with the following environment variables:
 
 |`*MONGODB_NOPREALLOC*`
 |Disable data file preallocation.
-|true
+|True
```
Comment (adellape) [inline using_images/db_images/mongodb.adoc]: @bparees Do you know whether these are case sensitive? It looks like a spec will accept it init-capped, at least.

Comment (bparees) [inline using_images/db_images/mongodb.adoc]: i don't know, we stuff the value in the mongo config here:
https://github.com/openshift/mongodb/blob/master/2.4/root/usr/share/container-scripts/mongodb/mongodb.conf.template

so it's up to mongo.  i'd guess mongo is not case sensitive.

Comment (adellape) [inline using_images/db_images/mongodb.adoc]: Thanks

**Comment by adellape** [comment]
@bfallonf LGTM. Up to you on the inline case sensitivity thing.

**Comment by bfallonf** [comment]
@adellape I changed to caps because it wasn't really about what's in the file, but it's the value. I can't imagine people are going to copy/paste one word...

But I changed to being parameter markup and pushed again.

Nothing of note changes. I'll merge away.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../using_images/db_images/mongodb.adoc#[Database Images -> MongoDB]
|Added the `*MONGODB_TEXT_SEARCH_ENABLED*` variable to the xref:../using_images/db_images/mongodb.adoc#environment-variables[Environment Variables] section.
%

**Pull request merged**